src/next-transpile-modules.js

```diff
@@ -60,7 +61,15 @@ const getPackageRootDirectory = (module) => {
 
     try {
       // Get the location of its package.json
-      packageRootDirectory = pkgDir.sync(packageDirectory);
+      const pkgPath = escalade(packageDirectory, (dir, names) => {
+        if (names.includes('package.json')) {
+          return 'package.json';
+        }
+        return false;
+      });
+      if (pkgPath != null) {
+        packageRootDirectory = path.dirname(pkgPath);
+      }
```

Comment:
I guess it should throw something is there is no `package.json` no?

Reply:
Just return `null`. AFAIK pkg-dir does not throw an error too.